modules/charon-core/src/main/java/org/wso2/charon3/core/protocol/endpoints/ResourceManager.java

```diff
@@ -352,4 +352,101 @@ default SCIMResponse updateWithPATCHRole(String id, String patchRequest, RoleV2M
         return new SCIMResponse(ResponseCodeConstants.CODE_NOT_IMPLEMENTED, ResponseCodeConstants.DESC_NOT_IMPLEMENTED,
                 Collections.emptyMap());
     }
+
+    /**
+     * POST method to create a new role in roleV2 model.
+     *
+     * @param postRequest Post request.
+     * @param roleManager RoleV2 manager.
+     * @return SCIMResponse.
+     */
+    default SCIMResponse createRoleMeta(String postRequest, RoleV2Manager roleManager) {
+
+        return new SCIMResponse(ResponseCodeConstants.CODE_NOT_IMPLEMENTED, ResponseCodeConstants.DESC_NOT_IMPLEMENTED,
+                Collections.emptyMap());
+    }
+
+    /**
+     * PUT method To update a role by giving entire attributes set.
```

Comment:
What is mean by entire attribute set here?

Reply:
I hope here we mean to update only role metadata. 

The keywords like only role metadata will imply that we are blocking something. We can indicate we are blocking user or group assignments or else can follow a pattern as suggested.